lib/suggestion-list-element.js

```diff
@@ -167,24 +159,7 @@ class SuggestionListElement extends HTMLElement {
     }
 
     atom.views.updateDocument(this.renderItems.bind(this))
-    return atom.views.readDocument(this.readUIPropsFromDOM.bind(this))
-  }
-
-  addActiveClassToEditor () {
```

Comment:
Because we add the active class to the editor above in the call stack, in the `showAtBeginningOfPrefix` and `showAtCursorPosition` methods, where we also create the overlay decoration.